pkgs/development/libraries/re2/default.nix

```diff
@@ -3,59 +3,46 @@
 , fetchFromGitHub
 , cmake
 , ninja
+, gtest
+, gbenchmark
+, abseil-cpp
+, icu
 , chromium
 , grpc
-, haskellPackages
 , mercurial
 , python3Packages
-, abseil-cpp
+, haskellPackages
```

Comment:
I don't understand the logic of this re-ordering of parameters.
I find alphabetical order the most simple, widely understood, logical order of all.
I wish we could just order parameters (automatically with a formatter) alphabetically and never have to second this anymore.

Reply:
Yeah, that works for me.